src/Training.TruckWorld.Clone.Backend/TruckWorld.Domain/Common/Entity.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace TruckWorld.Domain.Common;
+
+/// <summary>
+/// Base class for entities with a unique identifier.
+/// </summary>
+public class Entity : IEntity
+{
+    public Guid Id { get; set; }
```

Comment:
Entity class should be an abstract class

Reply:
why?